src/main/java/net/syscon/elite/api/resource/OffenderMovementsResource.java

```diff
@@ -5,20 +5,13 @@
 import io.swagger.annotations.ApiParam;
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
-import net.syscon.elite.api.model.CourtHearing;
-import net.syscon.elite.api.model.CourtHearings;
-import net.syscon.elite.api.model.ErrorResponse;
-import net.syscon.elite.api.model.PrisonToCourtHearing;
+import net.syscon.elite.api.model.*;
```

Comment:
I think agreement for team going forwards is no wildcard imports.

Reply:
Well... the Intellij settings that Peter displayed said to replace 5 or more imports with a *.  
![image](https://user-images.githubusercontent.com/58170926/77534325-57773e80-6e90-11ea-9fc6-1f22d81e63db.png)

Reply:
But if it's GDS policy then agreed - just change the class count in the screenshot above to 999.